### ISSUE TITLE
Add html entities escaping in quick doc view

### ISSUE THREAD

**Issue by grossws** [intellij-rust/intellij-rust#495]
Currently quick doc sometimes omits angle brackets, e.g. in doc for AsRef:

```
Both String and &str implement AsRef<str>:
fn is_hello  >(s: T) {
assert_eq!("hello", s.as_ref());
}
```

instead of `fn is_hello<T: AsRef<str>>(s: T) {`

**Comment by matklad** [comment]
If any one wants to look into this, here are the [tests] and the [code].

[tests]: https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/ide/docs/RustQuickDocumentationTest.kt

[code]: https://github.com/intellij-rust/intellij-rust/blob/master/src/main/kotlin/org/rust/ide/docs/RustDocumentationProvider.kt

**Comment by jajakobyly** [comment]
I will probably fix this with a reincarnation of #491

**Comment by grossws** [comment]
Issue still present in 0.0.1.758.

**Comment by matklad** [comment]
Yes, we've only recently merged the PR with the fix, the current nightly does not include it.

**Comment by grossws** [comment]
Ok, I'll test it on next plugin update.

**Comment by jajakobyly** [comment]
Ouch I believe #504 didn't fix it. I thought GitHub omits striked _fixed_ phrases :sob:

**Comment by matklad** [comment]
I think it will be autofix if we switch to https://github.com/valich/intellij-markdown, and it should be published to bintray soon.

**Comment by matklad** [comment]
Looks like ^^^^ has happened: https://github.com/valich/intellij-markdown/pull/15
